actions/deploy-tag-update/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -eu
+
+cd datarepo-helm-definitions
+
+# determine whether repo is api or ui
+case "${PWD}" in
```

Comment:
Using the directory name should be fine but it bugs me a little as you can git clone a repo into any directory name. Maybe using something like ` git config --get remote.origin.url` would be better?

Reply:
Well, I'm confused by this. The script uses `cd` above so wouldn't `PWD` always be `.../datarepo-helm-definitions` ?

Reply:
Yes, you're absolutely right! I will fix this. Thank you!